ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -45,6 +45,13 @@ cat << EOF > $SHARED_DIR/bastion_inventory
 ${BASTION_IP} ansible_ssh_user=centos ansible_ssh_common_args="$COMMON_SSH_ARGS" ansible_ssh_private_key_file="${SSH_PKEY}"
 EOF
 
+ADDITIONAL_ARG=""
+if [[ "$T5CI_JOB_TYPE" == "cnftests" ]]; then
+  ADDITIONAL_ARG="--cluster-name cnfdc2 --force"
+else
+  ADDITIONAL_ARG="-e $CL_SEARCH"
```

Comment:
you don't want to allocate cnfdc2 to anyone as well, so you need to use `--exclude cnfdc2`
Maybe cnfdc2 should put into variable